### PR TITLE
Updated map bounding box order for community app

### PR DESCRIPTION
**Pull request by ywkim312**

### CODE REVIEW

File: playbook-configs/joplin/map.config.json [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-	37.377222,
+	36.816347,
 	-94.192626,
-	36.816347
+	37.377222
   ],
   "ZOOM": 5,
   "MIN_ZOOM": 4,
-  "MAX_ZOOM": 19,
+  "MAX_ZOOM": 15,
```
Comment (longshuicy) [inline playbook-configs/joplin/map.config.json:17]: This should be 19?

File: playbook-configs/galveston/map.config.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+	27.941297,
 	-93.611436,
-	27.941297
+	30.541297
   ],
   "MAP_CENTER": [-94.911436, 29.241297],
   "ZOOM": 5,
   "MIN_ZOOM": 4,
-  "MAX_ZOOM": 19,
+  "MAX_ZOOM": 15,
```
Comment (longshuicy) [inline playbook-configs/galveston/map.config.json:11]: This needs to be 19

File: playbook-configs/slc/map.config.json [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+	41
   ],
   "MAP_CENTER": [
 	-111.954,
 	40.683
   ],
   "ZOOM": 5,
   "MIN_ZOOM": 4,
-  "MAX_ZOOM": 19,
+  "MAX_ZOOM": 15,
```
Comment (longshuicy) [inline playbook-configs/slc/map.config.json:20]: Same

**Review by longshuicy — CHANGES_REQUESTED** [review]

File: playbook-configs/galveston/map.config.json (hunk 2 above)
Comment (ywkim312) [reply playbook-configs/galveston/map.config.json]: updated

File: playbook-configs/slc/map.config.json (hunk 3 above)
Comment (ywkim312) [reply playbook-configs/slc/map.config.json]: updated

**Review by longshuicy — APPROVED** [review]

**Review by navarroc — APPROVED** [review]

**Pull request merged**